lms/views/api/canvas/sync.py

```diff
@@ -15,21 +15,23 @@ def __init__(self, request):
         permission=Permissions.API,
     )
     def sync(self):
+        course = self.get_course(self._request.json["course"]["context_id"])
         grading_user_id = self._request.json.get("learner", {}).get("canvas_user_id")
```

Comment:
I'm not sure how we intend to unify these things, but it feels like we are making life hard for ourselves by keeping `canvas_user_id` in the name.

I'm not sure of the full picture here, but it feels like we should have a standard field name and then modify how it works in the Canvas situtation?